tests/foreman/sys/test_fam.py

```diff
@@ -71,3 +120,20 @@ def test_positive_import_run_roles(sync_roles, target_sat):
     target_sat.cli.Host.ansible_roles_assign({'ansible-roles': roles, 'name': target_sat.hostname})
     play = target_sat.cli.Host.ansible_roles_play({'name': target_sat.hostname})
     assert 'Ansible roles are being played' in play[0]['message']
+
+
+@pytest.mark.e2e
+@pytest.mark.parametrize('ansible_module', FAM_TEST_PLAYBOOKS)
+def test_positive_run_modules_and_roles(module_target_sat, setup_fam, ansible_module):
+    """Run all FAM modules and roles on the Satellite
+
+    :id: b595756f-627c-44ea-b738-aa17ff5b1d39
+
+    :expectedresults: All modules and roles run successfully
+    """
+    # Execute test_playbook
+    result = module_target_sat.execute(
+        'export ANSIBLE_NOCOLOR=True && export PY_COLORS=0 && '
```

Comment:
Both [Ansible](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#ansible-nocolor) and [Pytest](https://docs.pytest.org/en/8.0.x/reference/reference.html#envvar-NO_COLOR) support [`NO_COLOR`](https://no-color.org/) so it should be sufficient to set `NO_COLOR=True`